### PR TITLE
made modal more responsive and added new modal

### PR DESCRIPTION
**Pull request by revengemiroz**

### CODE REVIEW

File: client/src/pages/Home/Home.js [hunk 1]
```diff
@@ -179,7 +179,7 @@ function Home() {
         onClose={closeModal}
       >
         <HeaderContainer>
-          <h1>Edit</h1>
+          <span>Edit</span>
           {/* eslint-disable-next-line */}
           <a onClick={closeModal}>X</a>
```
Comment (ivanms1) [inline client/src/pages/Home/Home.js]: Make this a button and remove styles, or create a custom button component where you can pass a prop to make not styled.

File: client/src/pages/Home/Home.js [hunk 2]
```diff
@@ -205,7 +205,7 @@ function Home() {
         onClose={closeDeleteModal}
       >
         <HeaderContainer>
-          <h1>Delete</h1>
+          <span>Delete</span>
           {/* eslint-disable-next-line */}
           <a onClick={closeDeleteModal}>X</a>
```
Comment (ivanms1) [inline client/src/pages/Home/Home.js]: same

File: client/src/pages/SubmitYourProject/Submit.js [hunk 3]
```diff
@@ -50,8 +61,8 @@ function Submit(props) {
           },
         },
       });
-      alert('success');
-      history.push('/');
+      img = data.preview;
+      await setOpen((o) => !o);
```
Comment (ivanms1) [inline client/src/pages/SubmitYourProject/Submit.js]: no need to await I believe, this is not a promise

File: client/src/pages/SubmitYourProject/Submit.js [hunk 4]
```diff
@@ -70,6 +81,26 @@ function Submit(props) {
...
+          <HeaderContainer>
+            <span>Project Submission</span>
+            <button onClick={closeModal}>&times;</button>
+          </HeaderContainer>
+          <Body>
+            <div className='imgContainer'>
+              <img src={img} alt={img}></img>
+            </div>
+            <p className='message'>Project have been submitted !</p>
+            <button onClick={() => history.push('/')}>Ok</button>
```
Comment (ivanms1) [inline client/src/pages/SubmitYourProject/Submit.js:100]: you could wrap the button inside a <Link /> component

File: client/src/pages/Home/Home.js (hunk 1 above)
Comment (revengemiroz) [reply client/src/pages/Home/Home.js]: for now changed the <a> tag to <button> tag will make this a component when i learn more

File: client/src/pages/Home/Home.js (hunk 2 above)
Comment (revengemiroz) [reply client/src/pages/Home/Home.js]: done

File: client/src/pages/SubmitYourProject/Submit.js (hunk 3 above)
Comment (revengemiroz) [reply client/src/pages/SubmitYourProject/Submit.js]: done

File: client/src/pages/SubmitYourProject/Submit.js (hunk 4 above)
Comment (revengemiroz) [reply client/src/pages/SubmitYourProject/Submit.js:100]: hope to be done in the next PR

**Review by ivanms1 — APPROVED** [review]

**Pull request merged**